SU-HSDO/suhumsci
# feat(STN-347):  Traditional Theme date stacked card

# READY FOR REVIEW

## Summary
Adds design for date tile on the Traditional Theme date stacked vertical card.

Designs: https://www.figma.com/file/bG42fKKioEcnJ0ORKkLL5E/%E2%9A%99%EF%B8%8F-Traditional-Theme---Working?node-id=5686%3A3

- [x] Design review completed
- [x] Add compiled CSS 

## Browser Testing

## Steps to Test
1. Ensure that `npm run test` passes in humsci_basic
2. Check the designs and verify that the date in the browser looks the same. As for Custom Econ Events views with this style, you can use: "Date Vertical - Grid of 3" or "Date Vertical - HP Grid"

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
